public/css/monitor.css

```diff
@@ -180,7 +173,7 @@ h3.sb-callout {
     min-height: 96vh;
   }
 
-  /* .landing-bottom-bar {
-    padding-top: 6vh;
-  } */
+  .landing-content .form-desc {
+    padding-bottom: 3rem;
```

Comment:
Just a note: at some point we should go thru and change our properties to [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties/Basic_concepts). But that's a bigger project.